src/harness/fake_sas.py

```diff
@@ -176,7 +176,15 @@ def _GetSuccessResponse(self):
   def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
-
+  def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
+    zoneData = req['zoneDate']
```

Comment:
This should be:

```
zone_data = request['zoneData']
```

1) We've been using snake_case (as opposed to camelCase) variable names in the rest of the code.
2) "req" is undefined so we use "request"
3) Typo: "zoneDate"